plugins/modules/packaging/os/rpm_ostree_pkg.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Dusty Mabe <dusty@dustymabe.com>
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2021, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: rpm_ostree_pkg
+short_description: Install or uninstall overlay additional packages
+version_added: "2.0.0"
+description:
+    - Install or uninstall overlay additional packages using C(rpm-ostree) command.
+options:
+    name:
+      description:
+      - Name of overlay package to install or remove.
+      required: true
+      type: list
+      elements: str
+      aliases: [ pkg ]
+    state:
+      description:
+      - State of the overlay package.
+      - C(present) simply ensures that a desired package is installed.
+      - C(absent) removes the specified package.
+      choices: [ 'absent', 'present' ]
+      default: 'present'
+      type: str
+author:
+- Dusty Mabe (@dustymabe)
+- Abhijeet Kasurde (@Akasurde)
```

Comment:
```suggestion
- Abhijeet Kasurde (@Akasurde)
notes:
- Does not support C(check_mode).
```

Reply:
isn't this by default.

Reply:
Most modules simply don't specify, and I think more people assume that check mode is supported by default. So I think it's better to be explicit.